bot/cogs/utils.py

```diff
@@ -250,8 +191,46 @@ async def vote(self, ctx: Context, title: str, *options: str) -> None:
         for reaction in options:
             await message.add_reaction(reaction)
 
-    async def send_pep_zero(self, ctx: Context) -> None:
-        """Send information about PEP 0."""
+    # PEPs area
+
+    @loop(hours=3)
+    async def refresh_peps_urls(self) -> None:
+        """Refresh PEP URLs listing in every 3 hours."""
+        # Wait until HTTP client is available
+        await self.bot.wait_until_ready()
+        log.trace("Started refreshing PEP URLs.")
+
+        async with self.bot.http_session.get(self.peps_listing_api_url) as resp:
+            listing = await resp.json()
+        log.trace("Got PEP URLs listing from GitHub API")
+
+        for file in listing:
+            name = file["name"]
+            if name.startswith("pep-") and name.endswith((".rst", ".txt")):
+                pep_number = name.replace("pep-", "").split(".")[0]
+                self.peps[int(pep_number)] = file["download_url"]
+        log.info("Successfully refreshed PEP URLs listing.")
```

Comment:
I find it harder to read a line that's outside and right under a block

```suggestion
                self.peps[int(pep_number)] = file["download_url"]

        log.info("Successfully refreshed PEP URLs listing.")
```